llvm/lib/Target/RISCV/RISCVISelLowering.cpp

```diff
@@ -89,8 +89,8 @@ RISCVTargetLowering::RISCVTargetLowering(const TargetMachine &TM,
   if ((ABI == RISCVABI::ABI_ILP32F || ABI == RISCVABI::ABI_LP64F) &&
       !Subtarget.hasStdExtF()) {
     errs() << "Hard-float 'f' ABI can't be used for a target that "
-                "doesn't support the F instruction set extension (ignoring "
-                          "target-abi)\n";
+              "doesn't support the F instruction set extension (ignoring "
```

Comment:
Please don't reformat unrelated code.

Reply:
ah, sorry. I had touched these lines in an earlier patch and hadn't realized I kept the formatting change. I'll fix this shortly.

Reply:
There is a lot of re-formatting to that file, not just the line I commented on.

Reply:
oh, I see it now.

Reply:
Sorry for that mess. I normally only use `git clang-format HEAD~`, but I must have use some other command or the `clangd` integration by mistake. I think this should be fixed. I'm looking at other files, and haven't seen anything obvious yet.

Reply:
Ah, I only use `git diff -U0 --no-color --relative main... -- | path/to/clang/tools/clang-format/clang-format-diff.py -p1  -i` :)

Reply:
Thanks. I'll try out that incantation tomorrow. :)